docs/concepts/dotnet-standard-library.md

```diff
@@ -0,0 +1,130 @@
+
+The spec is called "The .NET Standard Library". The more general ecosystem that builds on top of the spec is referred to as ".NET Standard" or coloquially as "NETStandard". A Library that targets the spec is called "a .NET Standard library (NSL)" or coloquially as "a NETStandard library" or as "an NSL" (analogue of "PCL").
+
+Specification
+=============
+
+The .NET Standard Library spec is a standardized set of APIs. A given .NET runtime will implement and advertise support for a particular version of the .NET Standard Library. That runtime is then able to support source and binaries that target that and previous versions of the spec.
+
+The spec is maintained by .NET runtime implementors, specifically Microsoft (includes .NET Framework, .NET Core and Mono) and Unity. A public feedback process is used as part of establishing new .NET Standard Library versions.
+
+Official Artifacts
+------------------
+
+The official spec is a set of .cs files that define the APIs that are part of the standard. The [ref directory](https://github.com/dotnet/corefx/tree/master/src/System.Runtime/ref) for each [component](https://github.com/dotnet/corefx/tree/master/src) defines the .NET Standard Library APIs. While the ref artifacts reside in the [CoreFX repo](https://github.com/dotnet/corefx), they are not .NET Core specific.
+
+The [NETStandard.Library](https://github.com/dotnet/corefx/blob/master/pkg/NETStandard.Library/NETStandard.Library.packages.targets) metapackage describes the set of libraries that define (in part) one or more .NET Standard Library versions.
+
+Derivative artifacts are provided to enable more convenient reading and to enable certain developer scenarios (for example, using a compiler).
+
+- API list in markdown (TBD)
+- Reference assemblies, distributed as [NuGet packages](../core-concepts/packages.md) and referenced by the [NETStandard.Library](https://www.nuget.org/packages/NETStandard.Library/) metapackage.
+
+Package Representation
+----------------------
+
+The primary distribution vehicle for the .NET Standard Library reference assemblies is [NuGet packages](../core-concepts/packages.md). Implementations will be delivered in a variety of ways, appropriate for each .NET runtime.
+
+NuGet packages target one of more frameworks. The .NET Standard Library packages target the ".NET Standard" framework, which is represented by the the `netstandard` Target Framework Moniker (TFM). The framework is ".NET Standard" instead of ".NET Standard Library", since ".NET Standard" includes both the .NET Standard Library packages and the larger ecosystem of libraries that are built on top of it. Libraries that are intended to run on multiple runtimes should use this target. 
```

Comment:
> The framework is ".NET Standard" instead of ".NET Standard Library", since ".NET Standard" includes both the .NET Standard Library packages and the larger ecosystem of libraries that are built on top of it.

I know what you want to say but that's virtually impossible to understand and even took me like three times before I actually got what you said. What about:

> The `netstandard` folder is meant to indicate the version of the .NET Standard Library that the component is compatible with. For packages that are part of the .NET Standard Library you can think of this folder name as indicating which version of .NET Standard Library it provides the support for.